# Copyright (c) 2024. Tudor Oancea, Matteo Berthet
std_msgs/Header header
float64 delta  # steering angle, rad
float64 tau_fl  # front left wheel torque, Nm
float64 tau_fr  # front right wheel torque, Nm
float64 tau_rl  # rear left wheel torque, Nm
float64 tau_rr  # rear right wheel torque, Nm
